### ISSUE TITLE
Farmer's Delight Refabricted incompatible with Connector Beta 42

### ISSUE THREAD

**Issue by RoomAlex68** [Sinytra/Connector#1008]
### Mod Name

Farmer's Delight Refabricted

### Mod Homepage

https://www.curseforge.com/minecraft/mc-mods/farmers-delight-refabricated

### Minecraft version

1.20.1

### Describe the bug

It is Incompatible with Connector Beta 42

### Steps to reproduce

![Screenshot_20240415_145913
(https://github.com/Sinytra/Connector/assets/85209177/60779ad8-c2c9-477f-98bb-918655bf77fd)

### Logs

I don't have a crash log

**Comment by SaltSouls** [comment]
I hate to say this, but why not just use the forge version? If I remember correctly, the refabricated version was specifically made to more accurately mimic the forge version of the mod. So in theory, the addons made for fabric that are compatible with refabricated should work with the forge version when using connector.

If not, then the more pressing issue should be getting those addons to work with connector and the forge version, rather than trying to get the fabric version to work. Likely in doing so, you will also get the fabric version to work as a side effect.

**Comment by SaltSouls** [comment]
> The fabric addons are incompatible with the forge version of farmer's delight. However, some of the fabric addons are being ported for the Refabricted version

Yes, I'm aware. The ones that are compatible with the original fabric version aren't compatible with the Refabricted version though. What I'm saying is that, in theory, the addons that are ported and made compatible with the Refabricted version, should be compatible with the forge version using connector as the Refabricted version was made to match the forge version as closely as possible code-wise. Again, if they are not, then rather than try and make the fabric version of FD compatible with forge using connector, it would be better to make the addons compatible with the forge version of FD instead.

Check [here](https://github.com/MehVahdJukaar/FarmersDelight/blob/1.20/information/Differences.md) for a full explanation of the differences between the original fabric version and the Refabricated version.

**Comment by RoomAlex68** [comment]
But I think it should be made compatible still on Connector

**Comment by RoomAlex68** [comment]
UPDATE ON THIS: 
I tested the Farmer's Delight Fabric addons that got ported to Refabricted, and they do not work on with Farmer's Delight Forge
Brewin and Chewin
Farmer's Respite 
Cultural Delights
Ube's Delight 
Frights Delight
Expanded Delight
Farmer's Knives
